src/ui/ExportToExcel/ExportToExcel.ts

```diff
@@ -131,14 +131,19 @@ export class ExportToExcel extends Component {
 
   private downloadExcelFile(content: ArrayBuffer) {
     const blob = new Blob([content], { type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' });
-    const url = URL.createObjectURL(blob);
-
-    const a = createAnchor();
-    a.href = url;
-    a.download = this.buildExcelFileName();
-    a.click();
-
-    URL.revokeObjectURL(url);
+    const fileName = this.buildExcelFileName();
+    if (window.navigator.msSaveOrOpenBlob) {
+      window.navigator.msSaveOrOpenBlob(blob, fileName);
+    } else {
```

Comment:
More a stylistic point, but I find it more readable to `return` after saving the blob with `msSaveOrOpenBlob` rather than adding an `else`.

Also, would it be possible to add a unit test for the `msSaveOrOpenBlob` path?